src/Modules/Shared/Microsoft.PowerShell.Host/Microsoft.PowerShell.Host.psd1

```diff
@@ -3,7 +3,7 @@ GUID="56D66100-99A0-4FFC-A12D-EEE9A6718AEF"
 Author="PowerShell"
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-linux`
Get-Module -ListAvailable.Get-Module -FullyQualifiedName @{ModuleName = 'Microsoft.PowerShell.Managemen*' ; ModuleVersion = '6.1.0.0'} - Wildcard
```powershell
Expected exactly '6.1.0.0', but got 7.0.0.0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Core/Get-Module.Tests.ps1: line 177
177:         $modules.Version | Should -BeExactly $ModuleVersion
```
Get-Module -ListAvailable.Get-Module -FullyQualifiedName @{ModuleName = 'microsoft.powershell.managemen*' ; ModuleVersion = '6.1.0.0'} - Mismatched case
```powershell
Expected exactly '6.1.0.0', but got 7.0.0.0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Core/Get-Module.Tests.ps1: line 177
177:         $modules.Version | Should -BeExactly $ModuleVersion
```